ui/nuxeo-document-preview.js

```diff
@@ -281,16 +281,20 @@ import './viewers/nuxeo-video-viewer.js';
         this.document.properties['vid:transcodedVideos'] &&
         this.xpath === 'file:content'
       ) {
-        const conversions = [];
-        this.document.properties['vid:transcodedVideos'].forEach((conversion) => {
-          if (conversion && conversion.content && conversion.content.data && conversion.content['mime-type']) {
-            conversions.push({
+        return this.document.properties['vid:transcodedVideos']
+          .filter(
+            (conversion) =>
+              conversion &&
+              conversion.content &&
+              conversion.content.data &&
+              conversion.content['mime-type'].match(/video.*/),
```

Comment:
nit: I think we should make sure that video is at the start of the string:
```suggestion
              conversion.content['mime-type'].match(/^video.*/),
```